src/chromeDebugAdapter.ts

```diff
@@ -352,10 +352,8 @@ export class ChromeDebugAdapter extends CoreDebugAdapter {
         this.events.emitStepStarted('LaunchTarget.LaunchExe');
         const platform = coreUtils.getPlatform();
         if (platform === coreUtils.Platform.Windows && shouldLaunchUnelevated) {
-            const semaphoreFile = path.join(os.tmpdir(), 'launchedUnelevatedChromeProcess.id');
-            if (fs.existsSync(semaphoreFile)) { // remove the previous semaphoreFile if it exists.
-                fs.unlinkSync(semaphoreFile);
-            }
+            const semaphoreFile = path.join(os.tmpdir(), utils.generateRandomString(8) + '.id');
```

Comment:
Is it possible that 2 different processes will generate the exact same file path? What will happen in that case?

Reply:
It's a extreme rare case, even rarer when on the same machine. I don't think we need to consider that.

Reply:
Let's add some telemetry for 15.8 just in case this ever happens.